ecosystem/sep-0031.md

```diff
@@ -68,6 +68,30 @@ All endpoints respond with content type:
 
 - `application/json`
 
+## Asset Conversions and Liquidity Strategies
+
+This protocol can be used if the Sending and Receiving Client wish to send and receive the same asset, but this is typically not the case. It is more common for the Sending Client to provide the Sending Anchor with a particular asset (the source asset) and have the Receiving Client receive a different asset (the destination asset).
+
+In this case, the source asset must convert into the receiving asset at some point prior to delivering funds to the Receiving Client. This conversion must be done by either the Sending Anchor, the Stellar Network, or the Receiving Anchor. The subsections below outline the stratgies used for each method.
+
+### Sending Anchor Asset Conversion
+
+The Sending Anchor can collect the source asset from the Sending Client and send the destination asset to the Receiving Anchor. 
+
+This approach requires the Sending Anchor to provide an exchange rate, or quote, to the Sending Client. It also requies the Sending Anchor to hold a balance of the destination asset on Stellar. The Sending Anchor would use Payment operations to send the quoted amount of the destination asset to the Receiving Anchor.
+
+### Stellar Network Asset Conversion
+
+Instead of holding a balance of the destination asset on Stellar, the Sending Anchor can instead us Path Payments to convert the source asset to the destination asset on the Stellar Network. 
+
+This approach requires a sufficiently liquid market to exist between the source and destination assets on the Stellar Decentralized Exchange (SDEX). Note that the exchange rate between the two assets is dynamic due to market fluctuations on the SDEX.
+
+### Receiving Anchor Asset Conversion
+
+Finally, the Receiving Anchor can receive payments of the source asset on Stellar and deliver payments of the destination asset to the Receiving Client. 
+
+This requires the Receiving Anchor to implement the [SEP-38 Anchor RFQ API](sep-0038.md), which enables the Receiving Anchor to provide quotes to the Sending Anchor. These quotes can be [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) or [firm](https://www.investopedia.com/terms/f/firmquote.asp). If a firm quote is used, the Stellar transaction _must_ be submitted to the Stellar network prior to the quote's expiration. More specifically, the `created_at` timestamp returned in the [Horizon GET /transactions/:id](https://developers.stellar.org/api/resources/transactions/) request must be earlier than the quote's expiration.
```

Comment:
💭 There are some subtleties regarding submitting transactions, the created at timestamp, and how long they are valid and can be resubmitted that we might need to consider in this scenario.

The transaction not only needs to be submitted before the quote's expiration, but it needs to be included in a closed ledger. The created at timestamp on transactions is the nomination time of the ledger (the time the ledger closed) that the transaction was included in. It will be possible for a transaction to be queued, and so it could be submitted prior to the expiry time but not included in the next ledger but rather a future ledger until after the close time. This may be confusing to implementers. This is an edge case, but we should probably still document what the two parties should do if this happens. Implementers could also reduce the chance of this happening by setting the time bounds on the transaction to be the expiry time or earlier.

Reply:
> Implementers could also reduce the chance of this happening by setting the time bounds on the transaction to be the expiry time or earlier.

If they do this, there is actually _no_ chance a transaction would be submitted earlier and included later than the expiration right? That detail may be important enough to explicitly mention here.

Reply:
I think it's a little fuzzy, I'd verify it with the core team, but essentially yes.